lib/context.js

```diff
@@ -979,6 +990,13 @@ function _findContextUrls(input, urls, replace, base) {
             // @context URL found
             urls[_ctx] = false;
           }
+        } else {
+          // look for scoped context
```

Comment:
@dlongley I think this handles retrieving scoped contexts properly when scanning for contexts, but please consider.

Reply:
Heh, I think tests would consider this better than a human. :)